### PR TITLE
Show application bindings on overview

### PR DESCRIPTION
**Pull request by spadgett**
https://trello.com/c/YRIHSS4W

Show what applications are bound to what provisioned services using pod presets.

openshift web console 2017-06-13 14-43-12

openshift web console 2017-06-13 14-42-28

cc @rhamilto @benjaminapetersen

### CODE REVIEW

File: app/styles/_overview.less [hunk 1]
```diff
@@ -81,6 +81,9 @@
       }
     }
   }
+  .service-binding h3 small {
```
Comment (rhamilto) [inline app/styles/_overview.less]: Nit:  rule is misalphabetized

File: app/views/overview/_service-instance-row.html [hunk 2]
```diff
@@ -81,30 +89,48 @@
         -->
         <div class="row">
           <div class="col-sm-12" ng-if="row.description">
-            <p class="pre-wrap" ng-bind-html="row.description | linky"></p>
+            <span class="pre-wrap" ng-bind-html="row.description | linky"></span>
```
Comment (rhamilto) [inline app/views/overview/_service-instance-row.html]: Semantically, a paragraph is a better choice here.

File: app/scripts/controllers/overview.js [hunk 3]
```diff
@@ -1093,6 +1095,66 @@ function OverviewController($scope,
...
+
+        // Create a selector for the potential binding target to check if the
+        // pod preset covers the selector.
+        var applicationSelector = new LabelSelector(_.get(apiObject, 'spec.selector'));
+        state.bindingsByApplicationUID[applicationUID] = [];
+
+        // Look at each pod preset selector to see if it covers this API object selector.
+        _.each(podPresetSelectors, function(podPresetSelector, bindingName) {
+          if (podPresetSelector.covers(applicationSelector)) {
+            // Keep a map of the traget UID to the binding and the binding to
```
Comment (jwforres) [inline app/scripts/controllers/overview.js]: nit, but typo on target

File: app/views/overview/_service-binding.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<div class="row service-binding">
+  <div class="col-sm-5 col-md-6">
+    <h3 ng-if="$ctrl.serviceClass">
+      {{$ctrl.serviceClass.externalMetadata.displayName || $ctrl.serviceClass.metadata.name}}
+      <small>{{$ctrl.binding.spec.instanceRef.name}}</small>
+    </h3>
+    <h3 ng-if="!$ctrl.serviceClass">
+      {{$ctrl.binding.metadata.name}}
```
Comment (jwforres) [inline app/views/overview/_service-binding.html]: think this was supposed to be updated to the instance name as well

File: app/styles/_overview.less (hunk 1 above)
Comment (spadgett) [inline app/styles/_overview.less]: Yeah I originally called it `.application-binding` and forgot to move when I renamed... Will fix.

File: app/views/overview/_service-binding.html (hunk 4 above)
Comment (spadgett) [inline app/views/overview/_service-binding.html]: Yes, thanks

**Comment by spadgett** [comment]
Thanks updated

**Comment by jwforres** [comment]
[merge]

**Pull request merged**